recipe/meta.yaml

```diff
@@ -16,7 +16,7 @@ requirements:
     - python
   run:
     - python
-    - analysis ==2021C1.2
+    - analysis ==2021C1.2=*_1
```

Comment:
What is "=*_1"?

Reply:
We want build [number 1](https://github.com/nsls-ii-forge/analysis-feedstock/pull/39/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR13), not 0. It should pick that file: https://anaconda.org/nsls2forge/analysis/2021C1.2/download/linux-64/analysis-2021C1.2-py37h1c21acf_1.tar.bz2. Since I was too lazy to specify the full build number prefix, I just used star for that. We want to pin explicitly here.